spec/features/users/user_signs_up_spec.rb

```diff
@@ -1,70 +1,53 @@
 
-  # Acceptance Criteria:
-  # * I must specify a valid email address,
-  #   password, and password confirmation
-  # * If I don't specify the required information, I am presented with
-  #   an error message
+# Acceptance Criteria:
+# * I must specify a valid email address,
+#   password, and password confirmation
+# * If I don't specify the required information, I am presented with
+#   an error message
 
-  # scenario 'provide valid registration information' do
-  #   visit new_user_registration_path
-  #
-  #   fill_in 'Username', with: 'mrappleseed'
-  #   fill_in 'Email', with: 'john@example.com'
-  #   fill_in 'Password', with: 'password'
-  #   fill_in 'Password confirmation', with: 'password'
-  #
-  #   click_button 'Sign up'
-  #
-  #   # expect(page).to have_content('Welcome! You have signed up successfully.')
-  #   expect(page).to have_content('Sign Out')
-  # end
-  #
-  # scenario 'provide invalid registration information' do
-  #   visit new_user_registration_path
-  #
-  #   click_button 'Sign up'
-  #   expect(page).to have_content("can't be blank")
-  #   expect(page).to_not have_content('Sign Out')
-  # end
-  #
-  # scenario 'username must be unique' do
-  #   existing_user = User.create(
-  #     username: 'username',
-  #     email: 'emailme@email.com',
-  #     password: '12345678',
-  #     password_confirmation: '12345678')
-  #
-  #   visit new_user_registration_path
-  #
-  #   fill_in 'Username', with: 'username'
-  #   fill_in 'Email', with: 'john@example.com'
-  #   fill_in 'Password', with: 'password'
-  #   fill_in 'Password confirmation', with: 'password'
-  #
-  #   click_button 'Sign up'
-  #
-  #   # expect(page).not_to have_content('Welcome! You have signed up successfully.')
-  #   expect(page).to have_content('Username has already been taken')
-  # end
-  #
-  # scenario 'password too short' do
-  #   visit new_user_registration_path
-  #
-  #   fill_in 'Username', with: 'username'
-  #   fill_in 'Email', with: 'john@example.com'
-  #   fill_in 'Password', with: '1'
-  #   fill_in 'Password confirmation', with: '1'
-  #
-  #   click_button 'Sign up'
-  #
-  #   # expect(page).not_to have_content('Welcome! You have signed up successfully.')
-  #   expect(page).to have_content('')
-  # end
+  scenario "provide valid registration information" do
+    visit new_user_registration_path
+
+    fill_in "Username", with: "mrappleseed"
+    fill_in "Email", with: "john@example.com"
+    fill_in "Password", with: "password"
+    fill_in "Password confirmation", with: "password"
+
+    click_button "Sign up"
+
+    expect(page).to have_content("Welcome! You have signed up successfully.")
+    expect(page).to have_content("Sign Out")
+  end
+
+  scenario "provide invalid registration information" do
+    visit new_user_registration_path
+
+    click_button "Sign up"
+
+    expect(page).to have_content("can't be blank")
+    expect(page).to_not have_content("Sign Out")
+  end
+  
+  scenario 'password too short' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.